packages/apps/kadena-docs/src/components/Layout/components/SideMenu/styles.ts

```diff
@@ -0,0 +1,177 @@
+import { styled, StyledComponent } from '@kadena/react-components';
+
+import Link from 'next/link';
+
+export const StyledSideMenu: StyledComponent<'div'> = styled('div', {
+  position: 'relative',
+  marginTop: '$6',
+});
+
+export const Menu: StyledComponent<
+  'div',
+  { isOpen?: boolean | 'true' | 'false' | undefined }
+> = styled('div', {
+  gridArea: 'menu',
+  position: 'absolute',
+  top: '$17',
+  height: 'calc(100% - $17 - $17)',
+  width: '100%',
+  borderRight: '1px solid $borderColor',
+  background: '$background',
+  overflow: 'hidden',
+  transform: 'translateX(-100%)',
+  transition: 'transform .3s ease, width .3s ease',
+
+  '@sm': {
+    width: '$leftSideWidth',
+  },
+  '@md': {
+    position: 'relative',
+    top: '0',
+    height: '100%',
+    transform: 'translateX(0)',
+  },
+
+  variants: {
+    isOpen: {
+      true: {
+        transform: 'translateX(0)',
+      },
+      false: {
+        transform: 'translateX(-100%)',
+        '@md': {
+          transform: 'translateX(0)',
+        },
+      },
+    },
+  },
+});
+
+export const StyledUl: StyledComponent<'ul'> = styled('ul', {
+  listStyle: 'none',
+  padding: 0,
+});
+
+export const StyledItem: StyledComponent<'li'> = styled('li', {
+  borderBottom: '1px solid $borderColor',
+  padding: '$4 0 $2',
+});
+
+export const StyledLink: StyledComponent<typeof Link> = styled(Link, {
```

Comment:
Is this reused a lot? I'm thinking we should have an Anchor component in the components library, but I see you're styling a next/link which makes that difficult. What do you think about this?